frontend/src/components/LandingPage/Features/Features.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import FeaturesCard from './FeaturesCard'
+import './Features.css'
+
+function Features()
+{
+    return
+    (
+        <div className = "featuresPage">
+            <h1 id = "header">App Features</h1>
+            <div className = "features">
+                <FeaturesCard
+                    CardImg = "Stock photo"
+                    CardTitle = "Explore nearby cities!"
+                    CardText = "Description"
+                    Button = "Explore now!"
+                />
+                <FeaturesCard
+                    CardImg = "Stock photo"
```

Comment:
All of the Cardimg props were giving me an error since the src isnt an actual photo. You can consider removing the need for the img if you would like.